sdk/schemaregistry/Azure.Data.SchemaRegistry/samples/Sample02_Serialization.md

```diff
@@ -0,0 +1,227 @@
+- [Next steps](#next-steps)
+- [Contributing](#contributing)
+
+## Implementing `SchemaValidator`
+
+The `SchemaValidator` is an abstract class that must be implemented and passed into the `SchemaRegistryJsonSerializer` constructor. This allows you to choose the third-party JSON Schema package you would like to use to generate schemas from .NET types and validate .NET objects against JSON schemas. Implementing both `SchemaValidator.Validate` and `SchemaValidator.GenerateSchema` is required.
+
+The following is an outline of how an implemented `SchemaValidator` may look:
+```C# Snippet:SampleSchemaRegistryJsonSchemaGeneratorImplementation
+internal class SampleJsonValidator : SchemaValidator<string>
+{
+    public override void Validate(Object data, Type dataType, string schemaDefinition)
+    {
+        // Your implementation using the third-party library of your choice goes here. This method throws
+        // an exception if the data argument is not valid according to the schemaDefinition.
+
+        List<Exception> validationErrors = SampleValidationClass.SampleValidationMethod(schemaDefinition, data, dataType);
+
+        if (validationErrors.Count  > 0)
+        {
+            throw new AggregateException(validationErrors);
+        }
+
+        return;
+    }
+
+    public override string GenerateSchema(Type dataType)
+    {
+        // Your implementation using the third-party library of your choice goes here.
+
+        return "<< SCHEMA GENERATED FROM DATATYPE PARAMETER >>";
+    }
+}
+```
+
+The following is a sample implementation:
+
+**This is for illustration only**
+```C#
+internal class SampleJsonValidator : SchemaValidator
+{
+    public override void Validate(object data, Type dataType, string schemaDefinition)
+    {
+        JSchema schema = JSchema.Parse(schemaDefinition);
+        JObject jsonObject = JObject.FromObject(data);
+
+        bool isValid = jsonObject.IsValid(schema, out IList<ValidationError> messages);
+        IEnumerable<Exception> ex = messages.Select((i => new Exception(i.Message)));
+
+        if (!isValid)
+        {
+            throw new AggregateException(ex);
+        }
+    }
+
+    public override string GenerateSchema(Type dataType)
+    {
+        JSchemaGenerator generator = new();
+        JSchema schema = generator.Generate(dataType);
+
+        return schema.ToString();
+    }
+}
+```
+
+## Serialize and deserialize JSON data using the Event Hub EventData model
+
+In order to serialize an `EventData` instance with JSON information, you can do the following:
+
+```C# Snippet:SchemaRegistryJsonSerializeEventData
+// The serializer serializes into JSON by default
+var serializer = new SchemaRegistrySerializer(client, groupName, new SampleJsonValidator());
+
+var employee = new Employee { Age = 42, Name = "Caketown" };
+EventData eventData = (EventData)await serializer.SerializeAsync(employee, messageType: typeof(EventData));
+
+// The schema Id will be included as a parameter of the content type
```

Comment:
nit:  These comments have no blank line before code and others do.  Either format is good, but we should consider being consistent.